src/models/constants.js

```diff
@@ -0,0 +1,31 @@
+const { model, Schema } = require('mongoose');
+const config = require('../config');
+
+const constantsSchema = new Schema({
+  _id: {
+    type: String,
+    required: true
```

Comment:
`_id` tem `required: true` por padrao 
```suggestion
    match: /^\d{17,19}$/,
```